airbyte-integrations/connectors/source-instagram/Dockerfile

```diff
@@ -12,5 +12,5 @@ RUN pip install .
 ENV AIRBYTE_ENTRYPOINT "python /airbyte/integration_code/main.py"
 ENTRYPOINT ["python", "/airbyte/integration_code/main.py"]
 
-LABEL io.airbyte.version=0.1.11
+LABEL io.airbyte.version=1.0.0
```

Comment:
Bumping to 1.0.0 because I think the logic of updating the state structure introduced one year ago can now be deprecated.